plugin/gcp/cloudFunction.go

```diff
@@ -0,0 +1,55 @@
+package gcp
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/puppetlabs/wash/plugin"
+	"google.golang.org/api/cloudfunctions/v1"
+)
+
+type cloudFunction struct {
+	plugin.EntryBase
+	service cloudFunctionsProjectService
+}
+
+func newCloudFunction(function *cloudfunctions.CloudFunction, service cloudFunctionsProjectService) *cloudFunction {
+	cf := &cloudFunction{
+		EntryBase: plugin.NewEntry(function.Name),
+		service:   service,
+	}
+	mtime, err := time.Parse(time.RFC3339, function.UpdateTime)
```

Comment:
Huh, only some services add a creation timestamp so it's in "Metadata".

Reply:
Yeah, it's weird.